src/styles/stars.css

```diff
@@ -0,0 +1,32 @@
+.star {
+  display: flex;
+}
+
+.star__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .star__star:nth-child(-n + 1) {
```

Comment:
Here you can use the comma to agroup different selectors that will use the same style.

For exemple

```
.stars—1 .star__star:nth-child(-n + 1),
.stars—2 .star__star:nth-child(-n + 2),
… {
  background-image: …
}
```